nvflare/private/fed/client/communicator.py

```diff
@@ -136,7 +136,7 @@ def client_registration(self, client_name, servers, project_name):
                 token = result.get_header(CellMessageHeaderKeys.TOKEN)
                 ssid = result.get_header(CellMessageHeaderKeys.SSID)
                 if not token and not self.should_stop:
-                    time.sleep(5)
+                    time.sleep(2)
```

Comment:
we might want to make this configurable

Reply:
I really don't see the meaningful reason for making this configurable. It's within the platform managed process. But made it configurable any way.